codegenerator/cli/templates/static/codegen/src/IO.res

```diff
@@ -104,19 +104,19 @@ let executeBatch = async (
       // they will only be in order per row, but not across the whole entity
       // table.
       let _ = entityHistoryItemsToSet->Js.Array2.sortInPlaceWith((a, b) => {
-        EventUtils.isEarlierEvent(
-          {
-            timestamp: a.current.block_timestamp,
-            chainId: a.current.chain_id,
-            blockNumber: a.current.block_number,
-            logIndex: a.current.log_index,
-          },
-          {
-            timestamp: b.current.block_timestamp,
-            chainId: b.current.chain_id,
-            blockNumber: b.current.block_number,
-            logIndex: b.current.log_index,
-          },
+        EventUtils.isEarlier(
+          (
+            a.current.block_timestamp,
```

Comment:
I'm a little bit worried that block handler events have timestamp as 0. @JonoPrest I think it might cause problems with history.